ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml

```diff
@@ -73,8 +65,8 @@ StatusListView {
     QtObject {
         id: d
 
-        property int sortBy: SortableTokenHoldersList.SortBy.None
-        property int sorting: SortableTokenHoldersList.Sorting.Descending
+        property int sortBy: TokenHoldersProxyModel.SortBy.None
+        property int sorting: Qt.DescendingOrder
```

Comment:
What about just `sortOrder`, to match https://doc.qt.io/qt-5/qt.html#SortOrder-enum

Reply:
done